src/libs/DateUtils.js

```diff
@@ -33,32 +34,35 @@ function getLocalMomentFromTimestamp(timestamp) {
  * Jan 20 at 5:30 PM          within the past year
  * Jan 20, 2019 at 5:30 PM    anything over 1 year ago
  *
+ * @param {String} locale
  * @param {Number} timestamp
  * @param {Boolean} includeTimeZone
  *
  * @returns {String}
  */
-function timestampToDateTime(timestamp, includeTimeZone = false) {
-    const date = getLocalMomentFromTimestamp(timestamp);
-    const isThisYear = moment().format('YYYY') === date.format('YYYY');
-    const isToday = moment().format('D MMM YYYY') === date.format('D MMM YYYY');
-    const yesterday = moment().add(-1, 'day').format('D MMM YYYY');
-    const isYesterday = yesterday === date.format('D MMM YYYY');
-
-    let format = 'LT';
-    if (isYesterday) {
-        format = `[Yesterday at] ${format}`;
-    } else if (!isToday) {
-        format = `MMM D [at] ${format}`;
-    } else if (!isThisYear) {
-        format = `MMM D, YYYY [at] ${format}`;
-    }
+function timestampToDateTime(locale, timestamp, includeTimeZone = false) {
+    const date = getLocalMomentFromTimestamp(locale, timestamp);
+    const TZ = '[UTC]Z';
```

Comment:
IMO it is clearer if you do this
```suggestion
    const TZ = includeTimeZone ? ' [UTC]Z' : '';
```
Then remove the `if (includeTimeZone) {` below and in each of the calendar key options do for example `[Today at] LT${TZ}` and remove the call to calendar that does not include ` ${TZ}`. 
Also, _I think_ that our guidelines say this should be named `tz`?

Reply:
Thank you for the suggestion, I made the requested change. Let me know if there is anything else.